src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1364,19 +1375,20 @@ namespace winrt::TerminalApp::implementation
 
             const auto controlConnection = _CreateConnectionFromSettings(realGuid, controlSettings);
 
-            const auto canSplit = focusedTab->CanSplitPane(splitType);
+            const float contentWidth = ::base::saturated_cast<float>(_tabContent.ActualWidth());
```

Comment:
huh, we don't end up with NAN anymore?

Reply:
i mean, during startup

Reply:
Yea, because even during startup, we're waiting for the page's first `LayoutUpdated` before we start processing arguments.